js/index.js

```diff
@@ -0,0 +1,106 @@
+// 1
+const formalgGreeting = 'Hello, ';
+const informalgGreeting = 'Hi, ';
+const exclamation = '!';
+const favouriteUser = 'Mark'
+
+const fullPhrase = function (inputName) {
+    if (inputName === favouriteUser){
```

Comment:
What if I pass lower cased word 'mark'. Seems that code will not work as expected. Please, add logic to ignore case sensitive issue